packages/node/src/ethereum/api.ethereum.ts

```diff
@@ -129,17 +129,35 @@ export class EthereumApi implements ApiWrapper<EthereumBlockWrapper> {
 
   async init(): Promise<void> {
     this.injectClient();
-    const [genesisBlock, network, supportsFinalization, supportsSafe] =
-      await Promise.all([
-        this.client.getBlock('earliest'),
-        this.client.getNetwork(),
-        this.getSupportsTag('finalized'),
-        this.getSupportsTag('safe'),
-      ]);
-    this.genesisBlock = genesisBlock;
-    this.supportsFinalization = supportsFinalization && supportsSafe;
-    this.chainId = network.chainId;
-    this.name = network.name;
+
+    try {
+      const [genesisBlock, network, supportsFinalization, supportsSafe] =
+        await Promise.all([
+          this.client.getBlock('earliest'),
+          this.client.getNetwork(),
+          this.getSupportsTag('finalized'),
+          this.getSupportsTag('safe'),
+        ]);
+
+      this.genesisBlock = genesisBlock;
+      this.supportsFinalization = supportsFinalization && supportsSafe;
+      this.chainId = network.chainId;
+      this.name = network.name;
+    } catch (e) {
+      //batch request error for filecoin
+      if (e.message.includes('"message":"we can\'t execute this request"')) {
```

Comment:
This would only work for filecoin. Other RPC providers might do other things. 

I think we should change this error handling. Maybe we can go off the http code or if its not a json rpc response